sys/include/suit.h

```diff
@@ -165,12 +177,25 @@ typedef struct {
     uint16_t offset;    /**< offset to the start of the content */
 } suit_param_ref_t;
 
+/**
+ * @name SUIT component flags.
+ *
+ * These state flags apply to individual components inside a manifest.
+ * @{
+ */
+#define SUIT_COMPONENT_STATE_FETCHED       0x01 /**< Component is fetched */
```

Comment:
should we keep one way of defining bits (1<<N) vs 0x01)?

Reply:
Do you have a preference for one or the other?

Reply:
no, just consistency. I suggest using the existing method for the new flags so we don't change unrelated code.

Reply:
Done!